### DISCUSSION TITLE
Function to mount/dismount a WIM file

### DISCUSSION THREAD

**Discussion by Tof06250** [PSAppDeployToolkit/PSAppDeployToolkit#704] [Ideas]
Hi everyone,
I'd like to start packaging big apps (with +2k files) in WIM format.
I'd love to be able to have a function that would mount this WIM file and start installation from the mount point then unmount.
This has been described here:
[Use a WIM to deploy large apps](https://adminsccm.com/2020/07/20/use-a-wim-to-deploy-large-apps-via-configmgr-app-model/)

I image I could simply use what's described here but one could perhaps add extra options to this function (that I don't have in mind while writing this).
Do you think it's possible?
Thanks

**Comment by That-Annoying-Guy** [comment]
You are in luck:

https://discourse.psappdeploytoolkit.com/t/psadt-extension-psadtwim/2888

**Reply by Tof06250** [reply]
Didn't find this, thanks for pointing me to it :)
And it's rather old... how did I miss this??

**Comment by mjr4077au** [comment]
This has been added natively to the upcoming v4 release.
